packages/@okta/vuepress-site/docs/reference/api/system-log/index.md

```diff
@@ -637,7 +637,7 @@ Bounded requests to the `/api/v1/logs` API have the following semantics:
 
 An expression filter is useful for performing structured queries where constraints on LogEvent attribute values can be explicitly targeted.
 
-The following expressions are supported for events with the `filter` query parameter:
+The following example expressions are supported for events with the `filter` query parameter:
```

Comment:
These are examples actually, not an exhaustive list. We should probably collapse the 2 sections.